demo/app/src/App.js

```diff
@@ -0,0 +1,349 @@
+import React, { useState, useEffect } from "react";
+import { Block } from "baseui/block";
+import { Button } from "baseui/button";
+import { Checkbox, STYLE_TYPE } from "baseui/checkbox";
+import { Input } from "baseui/input";
+import { Slider } from "baseui/slider";
+import { StatefulTabs, Tab } from "baseui/tabs";
```

Comment:
You can slim these down big time like this:

`import { Block, Button, Checkbox, STYLE_TYPE, Input, Slider, StatefulTabs, Tab } from "baseui"; = 75.4k`
Vs
```
import { Block } from "baseui/block"; = 45.7k
import { Button } from "baseui/button"; = 49.3k
import { Checkbox, STYLE_TYPE } from "baseui/checkbox"; = 52.8k
import { Input } from "baseui/input"; = 68.9k
import { Slider } from "baseui/slider"; = 58.4k
import { StatefulTabs, Tab } from "baseui/tabs"; = 53k
```